jsk_recognition_utils/requirements.txt

```diff
@@ -0,0 +1,2 @@
+chainer>=4.2.0
```

Comment:
Do we need this latest chainer?

Reply:
`catkin run_tests` passed with chainer>=2.0.0, so updated.